content/developer/vanilla-cli/build-process-v1.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Build Process - 1.0
+title: Build Process - v1
```

Comment:
If you're renaming "1.0" to "v1" you need a blurb somewhere on this page saying they're equivalent. Never delete information, our docs are meant to be relevant for any version.